src/main/asciidoc/json-pointers.adoc

```diff
@@ -0,0 +1,22 @@
+== Json Pointers
+
+Vert.x provides an implementation of <a href="https://tools.ietf.org/html/rfc6901">Json Pointers from RFC6901</a>.
+You can use pointers both for querying and for writing. You can build your {@link io.vertx.core.json.pointer.JsonPointer} using
+a string, a URI or manually appending paths:
+
+[source,java]
+----
+{@link examples.JsonPointerExamples#example1Pointers}
+----
+
+After instantiating your pointer, you can use method {@link io.vertx.core.json.pointer.JsonPointer#queryJson(java.lang.Object)} to query
+a JsonObject/JsonArray. If you want to write inside a JsonObject/JsonArray use {@link io.vertx.core.json.pointer.JsonPointer#writeJson(java.lang.Object, java.lang.Object)}:
+
+[source,java]
+----
+{@link examples.JsonPointerExamples#example2Pointers}
+----
+
+Vert.x Json Pointer algorithm abstracts from the queried data structure. That means that using {@link io.vertx.core.json.pointer.JsonPointer}
```

Comment:
_You can reuse Vert.x Json Pointer with custom object models too by providing a custom implementation of {@link io.vertx.core.json.pointer.JsonPointerIterator}._